cmake/InfluxDBConfig.cmake.in

```diff
@@ -1,11 +1,12 @@
 @PACKAGE_INIT@
 
 set(InfluxDB_VERSION @PROJECT_VERSION@)
+set(InfluxDB_WITH_BOOST @Boost_FOUND@)
 
 get_filename_component(InfluxDB_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 include(CMakeFindDependencyMacro)
 
-if(@Boost_FOUND@)
+if(InfluxDB_WITH_BOOST)
```

Comment:
More readable :+1:

Reply:
Actually, it is also a correctness fix, because `if(TRUE)` is deprecated / has to be controlled by a policy.

Reply:
Oh … second thing I learned about CMake by this PR.